packages/database/src/migrations/20221006004520-AddEntityRelationsDataTable-modifies-data.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+import { insertObject, generateId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  return insertObject(db, 'data_table', {
+    id: generateId(),
+    code: 'entity_relations',
+    description: 'Fetches data for entity relations',
+    type: 'internal',
+    permission_groups: '{*}',
```

Comment:
Safe to have public permissions for these data-tables, since the entity-api does permissions checking itself